indico/legacy/webinterface/tpls/RoomBookingNewBookingPeriodWidget.tpl

```diff
@@ -269,6 +323,10 @@ ${ form.repeat_interval(type='hidden') }
             checkFrequency();
         });
 
+        $('body').on('click', '.ui-multiselect-checkboxes :checkbox, .ui-multiselect-all, .ui-multiselect-none', function() {
```

Comment:
Do we really need to bind this on `body`? There's nothing more specific that doesn't change?